[2024-11-27T20:16:02.572+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-27T20:16:02.602+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_games_etl_daily.extract_data manual__2024-11-27T20:16:00.256947+00:00 [queued]>
[2024-11-27T20:16:02.607+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_games_etl_daily.extract_data manual__2024-11-27T20:16:00.256947+00:00 [queued]>
[2024-11-27T20:16:02.608+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-27T20:16:02.615+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): extract_data> on 2024-11-27 20:16:00.256947+00:00
[2024-11-27T20:16:02.621+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_games_etl_daily', 'extract_data', 'manual__2024-11-27T20:16:00.256947+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/daily_data_etl.py', '--cfg-path', '/tmp/tmpwkm0wi46']
[2024-11-27T20:16:02.622+0000] {standard_task_runner.py:91} INFO - Job 49: Subtask extract_data
[2024-11-27T20:16:02.623+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1580) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-27T20:16:02.624+0000] {standard_task_runner.py:63} INFO - Started process 1583 to run task
[2024-11-27T20:16:02.653+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_games_etl_daily.extract_data manual__2024-11-27T20:16:00.256947+00:00 [running]> on host 105fc369984c
[2024-11-27T20:16:02.711+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nba_games_etl_daily' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-27T20:16:00.256947+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-27T20:16:00.256947+00:00'
[2024-11-27T20:16:02.712+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-27T20:16:02.713+0000] {logging_mixin.py:188} INFO - Logical date: 2024-11-27 20:16:00.256947+00:00
[2024-11-27T20:16:02.713+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-27T20:16:02.713+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/daily_data_etl.py", line 47, in extract_data
    previous_day = get_previous_day(logical_date)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/daily_data_etl.py", line 25, in get_previous_day
    date = datetime.strptime(input_date, "%Y-%m-%d")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: strptime() argument 1 must be str, not DateTime
[2024-11-27T20:16:02.723+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_games_etl_daily, task_id=extract_data, run_id=manual__2024-11-27T20:16:00.256947+00:00, execution_date=20241127T201600, start_date=20241127T201602, end_date=20241127T201602
[2024-11-27T20:16:02.732+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 49 for task extract_data (strptime() argument 1 must be str, not DateTime; 1583)
[2024-11-27T20:16:02.757+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-11-27T20:16:02.764+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
